articles/hdinsight/hbase/apache-hbase-phoenix-squirrel-linux.md

```diff
@@ -26,13 +26,13 @@ Learn how to use [Apache Phoenix](http://phoenix.apache.org/) in Azure HDInsight
 ### Prerequisites
 Before you can use SQLLine, you must have the following items:
 
-* **An HBase cluster in HDInsight**. To create one, see [Get started with Apache HBase in HDInsight](./apache-hbase-tutorial-get-started-linux.md).
+* **An Apache HBase cluster in HDInsight**. To create one, see [Get started with Apache HBase in HDInsight](./apache-hbase-tutorial-get-started-linux.md).
 
-When you connect to an HBase cluster, you need to connect to one of the ZooKeeper VMs. Each HDInsight cluster has three ZooKeeper VMs.
+When you connect to an HBase cluster, you need to connect to one of the [Apache ZooKeeper](https://zookeeper.apache.org/) VMs. Each HDInsight cluster has three ZooKeeper VMs.
 
 **To get the ZooKeeper host name**
 
-1. Open Ambari by browsing to **https://\<cluster name\>.azurehdinsight.net**.
+1. Open [Apache Ambari](https://ambari.apache.org/) by browsing to **https://\<cluster name\>.azurehdinsight.net**.

```

Comment:
Why are we adding links everywhere? I don't think this is necessarily helpful, but is it required?

Reply:
Thank you for the feedback, I will discontinue adding the links.